client/src/store/modules/visit.js

```diff
@@ -93,7 +94,7 @@ const mutations = {
     if(debug) console.log('visit store top level menu ' + (top ? top : 'empty'))
     state.topLevelMenu = top
   },
-  setIsReadOnlyInstructor: (val) => {
+  setIsReadOnlyInstructor: (state, val) => {
```

Comment:
I see. However, `state` is needed here as a placeholder parameter, otherwise, we'd get bug and we wouldn't get the `val` since it was reflecting the state.